tests/staticfiles_tests/test_management.py

```diff
@@ -391,6 +391,17 @@ def test_ordering_override(self):
 
         self.assertFileContains('file2.txt', 'duplicate of file2.txt')
 
+    @override_settings(STATICFILES_STORAGE='django.contrib.staticfiles.storage.ManifestStaticFilesStorage')
+    def test_run_collectstatic_manifest_unchanged(self):
+        self.run_collectstatic(dry_run=False)
+        with open(os.path.join(settings.STATIC_ROOT, 'staticfiles.json'), 'r') as file:
+            manifest_before = file.read()
+
+        super().run_collectstatic(dry_run=True)
+        with open(os.path.join(settings.STATIC_ROOT, 'staticfiles.json'), 'r') as file:
+            manifest_after = file.read()
+        self.assertEqual(manifest_before, manifest_after)
```

Comment:
I think we can simply reuse existing `TestCollectionDryRun` instead of creating a separate test, e.g.:
```python
@override_settings(STATICFILES_STORAGE='django.contrib.staticfiles.storage.ManifestStaticFilesStorage')
class TestCollectionDryRunManifestStaticFilesStorage(TestCollectionDryRun):
    pass
```

Reply:
That makes sense. 
I tested locally and pushed the change.